python/benchmarks/basic_functions.py

```diff
@@ -232,7 +241,42 @@ class ModificationFunctions:
     params = PARAMS
     param_names = PARAM_NAMES
 
+    class LargeAppendDataModify:
+        """
+            This class will hold a cache of append large dataframes
+            The purpose of this cache is to create dataframes
+            which timestamps are sequenced over time so that 
+            overlap does not occur
+        """
+
+        def __init__(self, num_rows_list:List[int], number_elements:int):
+            self.df_append_large = {}
+            self.df_append_short_wide = {}
+            start_time = time.time()
+            for rows in num_rows_list:
+                print("Generating dataframe with rows: ", rows)
+                lst = list()
+                lst_saw = list()
+                for n in range(number_elements+1):
+                    print("Generating dataframe no: ", n)
+                    
+                    df = generate_pseudo_random_dataframe(rows, "s", get_time_at_fraction_of_df(2*(n+1), rows))
+                    df_saw = generate_random_floats_dataframe_with_index(
+                        ModificationFunctions.WIDE_DF_ROWS, ModificationFunctions.WIDE_DF_COLS, "s", 
+                        get_time_at_fraction_of_df(2*(n+1), rows=ModificationFunctions.WIDE_DF_ROWS)
+                    )
+
+                    lst.append(df)
```

Comment:
It looks like you're appending the dataframes in increasing order of timestamps.

But below you're `pop`ing them in decreasing order of timestamps. Don't we want to append them in increasing order?

Reply:
fixed